### ISSUE TITLE
Use of outdated task execution handler

### ISSUE THREAD

**Issue by hajekj** [dhilmathy/update-manifest-extension#1]
Hi, hope you are doing well.

We are using your task and are getting these warnings - do you plan to update the library or would you like me to fix it and submit PR?

```
##[warning]Task 'update-manifest' (1.1.7) is using deprecated task execution handler. The task should use the supported task-lib: https://aka.ms/tasklib
```

**Comment by dhilmathy** [comment]
@hajekj please feel free to submit PR if you have it fixed. Thanks!

**Comment by dhilmathy** [comment]
@hajekj. Thanks for reporting this issue. That was because of older PowerShell execution handler used. I'd upgraded that to `PowerShell3` handler and you shouldn't be getting this warning now. Released version [2.0.5](https://marketplace.visualstudio.com/items?itemName=dhilmathy.update-manifest) including this change. 

Fixed in PR #4
